apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html

```diff
@@ -82,6 +82,7 @@
                             <div class="inner-container">
                                 <div
                                     class="show-on-small-screen"
+                                    [ngClass]="{ 'pending-sales-balance-due': isPendingSales }"
```

Comment:
please check this class is added

Reply:
@aditi-tiwari17 yes, Shalinee has added this class in SCSS for responsive layout, but the class was not mentioned in HTML